plugin/pkg/scheduler/scheduler.go

```diff
@@ -140,26 +145,26 @@ func NewFromConfigurator(c Configurator, modifiers ...func(c *Config)) (*Schedul
 }
 
 // Run begins watching and scheduling. It starts a goroutine and returns immediately.
-func (s *Scheduler) Run() {
-	go wait.Until(s.scheduleOne, 0, s.config.StopEverything)
+func (sched *Scheduler) Run() {
```

Comment:
I'm not even sure how to check the version heh. I'm just install golint with `go get -u github.com/golang/lint/golint`